### PR TITLE
[Helm] Add .Values.revisionHistoryLimit

### PR DESCRIPTION
**Pull request by jjaniec**
Hi ! Just a small PR to allow us to customize the revisionHistoryLimit of the agents Deployment, thanks for your work :)

### CODE REVIEW

File: src/helm/blue-agent/templates/deployment.yaml [hunk 1]
```diff
@@ -10,6 +10,7 @@ spec:
     matchLabels:
       {{- include "blue-agent.selectorLabels" . | nindent 6 }}
   replicas: {{ .Values.replicaCount | int | required "A value for .Values.replicaCount is required" }}
+  revisionHistoryLimit: {{ .Values.revisionHistoryLimit }}
```
Comment (clemlesne) [inline src/helm/blue-agent/templates/deployment.yaml:13]: Missing `int` cast, `required` constraint and and custom error message (like the above line).

File: src/helm/blue-agent/values.yaml [hunk 2]
```diff
@@ -126,6 +126,9 @@ tolerations: []
 # Pod affinity configuration
 affinity: {}
 
+# Number of revisions to keep in the history of the Deployment
```
Comment (clemlesne) [inline src/helm/blue-agent/values.yaml:129]: Value to `0` will cause the inability to rollback. Can you specify the documentation link as note plus a warning?

https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#clean-up-policy

**Comment by jjaniec** [comment]
You're right, it's better now :)

**Review by clemlesne — APPROVED** [review]
👌🏻

**Pull request merged**